apps/gitness/src/components/FileViewer.tsx

```diff
@@ -13,12 +24,37 @@ import { useGetRepoRef } from '../framework/hooks/useGetRepoPath'
 import { getTrimmedSha, normalizeGitRef } from '../utils/git-utils'
 import { PathParams } from '../RouteDefinitions'
 import { timeAgoFromISOTime } from '../pages/pipeline-edit/utils/time-utils'
+import FileContentViewer from './FileContentViewer'
+
+interface PathParts {
+  path: string
+  parentPath: string
+}
+function splitPathWithParents(fullResourcePath: string) {
```

Comment:
can we move this funct to a utils function?

Reply:
sure